lib/interceptor.js

```diff
@@ -55,9 +55,24 @@ Interceptor.prototype.reply = function reply(statusCode, body, headers) {
     if (arguments.length <= 2 && _.isFunction(statusCode)) {
         body = statusCode;
         statusCode = 200;
-    }
 
-    this.statusCode = statusCode;
+        this.statusCode = statusCode;
+    }
+    else if (_.isObject(statusCode)) {
+        var statusObject = {}
+        for (var prop in statusCode) {
+            if (prop === 'statusCode') {
+                this.statusCode = statusCode[prop]
+            }
+            else {
+                statusObject[prop] = statusCode[prop]
+            }
+        }
+        this.statusObject = statusObject
```

Comment:
Instead of setting the whole `statusObject` to the interceptor, I suggest setting just an additional `statusMessage`. This way, subsequent code (`RequestOverrider`) does not need to know of the interface of `Interceptor.reply`.